device-types/Cisco/MDS-C9148-16P.yaml

```diff
@@ -0,0 +1,120 @@
+---
+manufacturer: Cisco
+model: MDS 9148 16P
+slug: ds-9148-16p
+part_number: DS-C9148-16P-K9
+u_height: 1
+is_full_depth: true
+power-ports:
```

Comment:
It looks like these power supplies are modular. Is that true? If so, then these should be inside of `module-bays` and the associated modular PSUs should be added to the module-types folder if you can